app/konnect/analytics/explorer.md

```diff
@@ -40,6 +44,100 @@ Kong latency | Latency | The amount of time, in milliseconds, that {{site.base_g
 Request Size | Size | The size of the request payload received from the client, in bytes. Users can select between different percentiles (p99, p95, and p50). For example, a 99th percentile request size of 100 bytes means that the payload size for every 1 in 100 requests was at least 100 bytes.
 Response Size | Size | The size of the response payload returned to the client, in bytes. Users can select between different percentiles (p99, p95, and p50). For example, a 99th percentile response size of 100 bytes means that the payload size for every 1 in 100 response back to the original caller was at least 100 bytes.
 
+{% endnavtab %}
+{% navtab Time intervals %}
+
+The time frame selector controls the time frame of data visualized, which indirectly controls the
+granularity of the data. For example, the “5M” selection displays five minutes in
+one-second resolution data, while longer time frames display minute, hour, or days resolution data.
+
+All time interval presets are **relative**. 
+For custom reports, you can also choose a **custom** date range.
+
+* **Relative** time frames are dynamic and the report captures a snapshot of data
+relative to when a user views the report.
+* **Custom** time frames are static and the report captures a snapshot of data
+during the specified time frame. You can see the exact range below
+the time frame selector. For example:
+
+    ```
+    Jan 26, 2023 12:00 AM - Feb 01, 2023 12:00 AM (PST)
+    ```
+
+Interval | Description  
+---------|-------------
+Last 15 minutes | Data is aggregated in one minute increments.
+Last hour| Data is aggregated in one minute increments.
+Last six hours | Data is aggregated in one minute increments.
+Last 12 hours| Data is aggregated in one hour increments.
+Last 24 hours| Data is aggregated in one hour increments.
+Last seven days | Data is aggregated in one hour increments.
+Last 30 days | Data is aggregated in daily increments.
+Current week | Data is aggregated in one hour increments. Logs any traffic in the current calendar week. 
+Current month | Data is aggregated in one hour increments. Logs any traffic in the current calendar month. 
+Previous week | Data is aggregated in one hour increments. Logs any traffic in the previous calendar week.
+Previous month | Data is aggregated in daily increments. Logs any traffic in the previous calendar month. 
+
+{% endnavtab %}
+{% endnavtabs %}
+
+## LLM usage reporting
```

Comment:
Where is the LLM info coming from? Does a user need to configure AI Gateway plugins before this data will start streaming into Konnect?

Reply:
Yes.